lib/editor/util/map.js

```diff
@@ -769,7 +769,8 @@ export function stopToPatternStop (
     dropOffType: 0,
     pickupType: 0,
     shapeDistTraveled: null,
-    timepoint: null
+    timepoint: null,
+    stopHeadsign: ''
```

Comment:
Sort attributes

Reply:
Sorted in https://github.com/ibi-group/datatools-ui/pull/890/commits/ddf848719158b3500fd6520202ec781d5302dd5d